server/src/test/java/io/spine/server/GrpcContainerTest.java

```diff
@@ -104,6 +103,40 @@ void startServerWithExecutor() throws IOException {
                 .isNotNull();
     }
 
+    @Test
+    @DisplayName("configure underlying gRPC server")
+    void configureUnderlyingGrpcServer() {
+        var port = 1654;
+        var service = CommandService
```

Comment:
Can we have `.newBuilder()` on the previous line, please? 

Probably, we need to have a test fixture which would read something like `CommandSerice.noOp()` or `CommandService.servingNoCommands()`.

Reply:
I have refrained from creating a fully-fledged fixture set, as we only have such cases thrice in two different tests. But I have created a new `Given...` on this matter.